corehq/motech/repeaters/optionvalue.py

```diff
@@ -0,0 +1,68 @@
+import attr
+
+
+class OptionValue(property):
+
+    NOT_SET = object()
+
+    def __init__(
+        self,
+        default=NOT_SET,
+        choices=None,
+        schema=None,
+        schema_list=None
+    ):
+        if schema and default is not self.NOT_SET:
+            raise ValueError("default not allowed with schema")
+        self.default = default
+        self.choices = choices
+        self.schema = schema
+        self.schema_list = schema_list
```

Comment:
Do we still need `schema` and `schema_list`? If not, what do you think of removing them since they have problems that we have not fully ironed out?

At least `schema_list` seems especially problematic. I don't see a problem with keeping `schema` since it has clearer and less surprising semantics.